telegram/commands/gemini/gemini.go

```diff
@@ -0,0 +1,28 @@
+package gemini
+
+import (
+	"activity-tracker/gemini"
+	"activity-tracker/shared"
+	"errors"
+	"fmt"
+)
+
+var (
+	errNoPrompt = errors.New("En serio metiste un prompt vacío bobo hijueputa? Haz algo bien sub-humano 🙄")
+	forceText   = "responde solo texto"
+)
+
+func HandleGemini(bot *shared.Bot, chatID int64, userName, content string) error {
```

Comment:
userName is not used, remove it

Reply:
we need to refactor (again) the code cuz there's a lot of func who has userName as param an never use it

Reply:
solved

Reply:
oki